doc/adr/index.rst

```diff
@@ -0,0 +1,24 @@
+Architectural Decision Records
+==============================
+
+This section stores the architectural decision records.
+
+From  `Architecture decision record (ADR) examples for software planning, IT leadership, and template documenation <joel_>`_, by Joel Parker Henderson:
```

Comment:
correct the spelling of "documenation" to "documentation" -- word is missing a "t"